systest/parameters/longfast/resources.env

```diff
@@ -0,0 +1,2 @@
+poet_resources={"requests": {"cpu": "1", "memory": "16Gi"}, "limits": {"cpu": "2", "memory": "16Gi"}}
```

Comment:
NIT: Poet can be configured to keep more layers of the tree on disk with `--memory=N` with a small N like even `0`. Perhaps we could save some $ in CI  by keeping the tree on disk and requesting less memory for poet Pods. The downside is that generation of the poet proof (Merkle proof, not the sequential work itself) will take longer.

Reply:
thanks, i will skip that